vector/src/main/res/values/config.xml

```diff
@@ -8,6 +8,7 @@
     <string name="matrix_org_server_url" translatable="false">https://matrix.org</string>
     <string name="default_hs_server_url" translatable="false">https://matrix.org</string>
     <string name="default_identity_server_url" translatable="false">https://vector.im</string>
+    <string name="piwik_server_url" translatable="false">"https://piwik.riot.im"</string>
```

Comment:
The final `/ ` has been lost in the operation, is it still working without?

Reply:
Yes, I had a look in the piwik source and it adds the `/` if it is not there.

Reply:
Ok, great, I've also checked it :)